modules/edu.gemini.seqexec.server/src/main/resources/DhsHdr.xml

```diff
@@ -0,0 +1,1046 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
This is the import of the Channel Access definition file used for the DHS keywords in the old Seqexec. Not yet used, but still included for future work.